test/bigtest/tests/Utils/order/show-update-order-error-test.js

```diff
@@ -6,30 +6,95 @@ import {
 import { expect } from 'chai';
 import sinon from 'sinon';
 
-import showUpdateOrderError from '../../../../../src/components/Utils/order/showUpdateOrderError';
-
-describe('showUpdateOrderError', () => {
-  let fakeCallout;
-  let fakeResponse;
-
-  beforeEach(async () => {
-    fakeCallout = {
-      sendCallout: sinon.spy(),
-    };
-    fakeResponse = {
-      json: () => ({
-        errors: [
-          {
-            code: 'vendorIsInactive',
-          },
-        ],
-      }),
-    };
-
-    await showUpdateOrderError(fakeResponse, fakeCallout);
+import { processOpenOrderError, ERROR_CODES } from '../../../../../src/components/Utils/order';
+
+const CALLOUT_ERROR_CODES =
+  Object.keys(ERROR_CODES).filter(item => item !== 'vendorIsInactive' && item !== 'vendorNotFound');
+
+describe('processOpenOrderError', () => {
+  CALLOUT_ERROR_CODES.forEach(errorCode => {
+    describe(`process callout for ${errorCode} error`, () => {
+      let fakeCallout;
+      let fakeResponse;
+      let fakeOpenModal;
+
+      beforeEach(async () => {
+        fakeCallout = {
+          sendCallout: sinon.spy(),
+        };
+        fakeResponse = {
+          json: () => ({
+            errors: [
+              {
+                code: errorCode,
+              },
+            ],
+          }),
+        };
+        fakeOpenModal = sinon.spy();
+
+        await processOpenOrderError(fakeResponse, fakeCallout, fakeOpenModal);
+      });
+
+      it('call callout with right error', () => {
+        expect(fakeCallout.sendCallout.firstCall.args[0].message.props.id).to.equal(`ui-orders.errors.${errorCode}`);
+      });
+    });
   });
 
-  it('should call callout with right error', () => {
-    expect(fakeCallout.sendCallout.firstCall.args[0].message.props.id).to.equal('ui-orders.errors.vendorIsInactive');
+  describe('process modal for vendorIsInactive', () => {
+    let fakeCallout;
+    let fakeResponse;
+    let fakeOpenModal;
+
+    beforeEach(async () => {
+      fakeCallout = {
+        sendCallout: sinon.spy(),
+      };
+      fakeResponse = {
+        json: () => ({
+          errors: [
+            {
+              code: 'vendorIsInactive',
+            },
+          ],
+        }),
+      };
+      fakeOpenModal = sinon.spy();
+
+      await processOpenOrderError(fakeResponse, fakeCallout, fakeOpenModal);
+    });
+
+    it('call open error modal with vendorIsInactive error code', () => {
+      expect(fakeOpenModal.args[0][0]).to.equal('vendorIsInactive');
+    });
+  });
+
+  describe('process modal for vendorNotFound', () => {
+    let fakeCallout;
+    let fakeResponse;
+    let fakeOpenModal;
+
+    beforeEach(async () => {
+      fakeCallout = {
+        sendCallout: sinon.spy(),
+      };
+      fakeResponse = {
+        json: () => ({
+          errors: [
+            {
+              code: 'vendorNotFound',
```

Comment:
please use constant from the code for error codes, in case if they are changed in a future

Reply:
rewrite with constants